panels/source/arrangers/Arranger.js

```diff
@@ -363,16 +363,20 @@
 				unit = unit || 'px';
 				if (!this.updating) {
 					// IE10 uses setBounds because of control hit caching problems seem in some apps
-					if (enyo.dom.canTransform() && !control.preventTransform && !enyo.platform.android && enyo.platform.ie !== 10) {
```

Comment:
For reference, this was the original commit that added the guard: https://github.com/enyojs/layout/commit/0f262b3204e3407aae500b9a9ef7266cfd62d6af